src/features/blocks/BlocksCategory.tsx

```diff
@@ -0,0 +1,36 @@
+import './BlocksCategory.css'
+import {ToolboxCategory} from "../../toolboxconfig"
+import Block from "./Block"
+
+function BlocksCategory(props: ToolboxCategory) {
+  const categoryStyle = {
+    borderLeft: `10px solid ${props.highlight}`
+  }
+
+  function toggleCategory(e: any) {
```

Comment:
Should be typed MouseEvent, cf https://fettblog.eu/typescript-react/events/

Reply:
After typing it as MouseEvent & typing the target (to prevent is possibly null), I'm having trouble figuring out how to solve the following warning:
![image](https://user-images.githubusercontent.com/7738915/115850483-2fa53200-a426-11eb-8f7a-88717b617d4c.png)

Any ideas? Bit stumped. @SebastienTainon

Reply:
You should typecast `categoryBlocksDiv` to `HTMLElement`. Cf for instance https://stackoverflow.com/questions/58773652/ts2339-property-style-does-not-exist-on-type-element

This is because not all elements are supposed to have a "style" property"